pages/linux/at.md

```diff
@@ -0,0 +1,11 @@
+# at
+
+> Executes commands at a specified time.
+
+- Open 'at' prompt to execute commands at specific time. When you done press Ctrl+D to register job(s):
```

Comment:
Suggestion: "Open \`at` prompt to type commands to be executed at the given time"

"When you done" --> "When you're done", and maybe also add a comma here, to make the reading flow more explicit.

Also, it's slightly unclear what "register job(s)" mean, since the terms aren't used before. How about "save the scheduling"?